api/boilerplate/views.py

```diff
@@ -17,3 +25,50 @@ class GroupViewSet(viewsets.ModelViewSet):
     """
     queryset = Group.objects.all()
     serializer_class = GroupSerializer
+
+# This function is based on django.contrib.auth.views.login
```

Comment:
For function comments, please use `"""` (cf [PEP257](https://www.python.org/dev/peps/pep-0257/))